lsp-mode.el

```diff
@@ -355,6 +355,8 @@ the server has requested that."
     "[/\\\\]build-aux\\'"
     "[/\\\\]autom4te.cache\\'"
     "[/\\\\]\\.reference\\'"
+    ;; Bazel
+    "bazel-[^/\\\\]+\\'",
```

Comment:
There is a `,` after the string? That should be removed

Reply:
Yep, sorry. Typo fixed.